client/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_SERVER_URL = 'http://localhost:3001'
```

Comment:
Looks good, maybe we can consider some more documentation on ports: https://dev.mysql.com/doc/mysql-port-reference/en/mysql-ports-reference-tables.html